doc/NEWS

```diff
@@ -125,6 +125,12 @@ Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
 - Add support for ssh new-style private keys encrypted using `aes256-ctr`
   cipher.  [vkhromov; 2020]
 
+- Implement full checking in SSH OpenCL format, and drop FMT_NOT_EXACT from
```

Comment:
`NEWS` is meant to be end-user friendly - we shouldn't mention things internal to source code in there. So instead of mentioning `FMT_NOT_EXACT`, we should explain that change's effect in plain English. We should also not mix unrelated things in one sentence like this (a major OpenCL-specific change, and an unrelated change to both formats).

Anyway, I suppose we'll need to go over `NEWS` and add/revise/re-group its entries before the next release.